packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Section.Example.tsx

```diff
@@ -53,7 +53,7 @@ export class ContextualMenuSectionExample extends React.Component<any, any> {
                 }
               },
               {
-                key: 'section',
+                key: 'section3',
```

Comment:
Lets leave 1 of these keys out to confirm that it works.

Reply:
@joschect Do you mean to catch a bug in a CI build? If so, I don't think it would be caught. It's a console warning only when using React in developer mode. It won't appear with a production version of React.